spec/system/users/edit_user_spec.rb

```diff
@@ -19,15 +22,10 @@
   feature 'being logged in' do
     before do
       sign_in(user)
+      edit_user_page.submit(new_email, password)
```

Comment:
I don't like having steps of a test in before blocks. If they are part of the test I want to see them. (this gets even worser when you have many tests and need to scroll to find this before block)